core/templates/components/graph-services/graph-layout.service.spec.ts

```diff
@@ -593,33 +585,33 @@ describe('Graph Layout Service', () => {
       }
     };
 
-    let graphWidthUpperBound = sgls.getGraphWidth(
+    let graphWidthUpperBoundPixel = sgls.getGraphWidth(
```

Comment:
No, this name suggests that it is a pixel.

Try graphWidthUpperBoundInPixels perhaps, or graphWidthUpperBoundInPx or graphWidthUpperBoundPx if that's too long. Ditto elsewhere.

Reply:
Only for graphWidthUpperBound or throughout the file?

Reply:
Done